R/teal_data_module.R

```diff
@@ -22,6 +22,10 @@
 #'  `shiny` module server function; must only take `id` argument;
 #'  must return reactive expression containing `teal_data` object
 #' @param label (`character(1)`) Label of the module.
+#' @param once (`logical(1)`) Applicable only when used in the `data` argument of `init`.
+#'  If `TRUE`, the data module can only be consumed once. After the successful consumption,
+#'  the data module tab will disappear and the user will not be able to interact with the UI of the module.
```

Comment:
Please also elaborate somewhere in a documentation about data-reloading. Separate section in this topic (`teal_data_module`) makes sense.

```suggestion
#'  If `TRUE`, the data module will be shown only once and will disappear after successful data loading. 
#' App user will no longer be able to interact with this module anymore. 
#' If `FALSE` ...
```

In the same time please get ready for a discussion (after merge) about `teal_data_module` and `teal_transform_module`. IMO we can't have a single constructors for both as assertions on their `server` function differ and now also `once` is dedicated to `teal_data_module` only.